src/Database.php

```diff
@@ -251,11 +247,10 @@ public function aggregate(array $pipeline, array $options = [])
      * @see DatabaseCommand::__construct() for supported options
      * @param array|object $command Command document
      * @param array        $options Options for command execution
-     * @return Cursor
      * @throws InvalidArgumentException for parameter/option parsing errors
      * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
      */
-    public function command(array|object $command, array $options = [])
+    public function command(array|object $command, array $options = []): Cursor
```

Comment:
Not a `CursorInterface`?
```suggestion
    public function command(array|object $command, array $options = []): CursorInterface
```

Reply:
Updated